daemon/algod/api/algod.oas2.json

```diff
@@ -3189,6 +3189,41 @@
         }
       }
     },
+    "SimulateRequest": {
+      "description": "Request type for simulation endpoint.",
+      "type": "object",
+      "required": [
+        "txn-groups"
+      ],
+      "properties": {
+        "txn-groups": {
+          "description": "The transaction groups to simulate.",
```

Comment:
How is it ambiguous if you say this 2,000 lines later

Reply:
Would you prefer if this indicated we only support 1 group for now?